tests/store/AnnexesTestCase.py

```diff
@@ -0,0 +1,58 @@
+from unittest.mock import patch
+
+from ignf_gpf_api.io.ApiRequester import ApiRequester
+from ignf_gpf_api.store.Annexes import Annexes
+from tests.GpfTestCase import GpfTestCase
+
+
+class AnnexesTestCase(GpfTestCase):
+    """Tests Annexes class.
+
+    cmd : python3 -m unittest -b tests.store.AnnexesTestCase
+    """
```

Comment:
Idem pour l’homogénéité à mettre au singulier.
```suggestion
class AnnexeTestCase(GpfTestCase):
    """Tests Annexe class.

    cmd : python3 -m unittest -b tests.store.AnnexeTestCase
    """
```